service/src/test/java/bio/terra/workspace/service/workspace/flight/azure/CreateAzureContextFlightTest.java

```diff
@@ -52,7 +77,7 @@ void successCreatesContext() throws Exception {
     // Wait for the job to complete
     FlightState flightState =
         StairwayTestUtils.pollUntilComplete(
-            jobId, jobService.getStairway(), Duration.ofSeconds(30), STAIRWAY_FLIGHT_TIMEOUT);
+            jobId, jobService.getStairway(), Duration.ofSeconds(1), STAIRWAY_FLIGHT_TIMEOUT);
```

Comment:
I changed all the polling intervals in this file to 1s. Each tests now takes only a few seconds instead of 30s.